src/Database/Connections/Connection.php

```diff
@@ -1,11 +1,28 @@
 <?php namespace Winter\Storm\Database\Connections;
 
+use Doctrine\DBAL\Connection as DoctrineConnection;
+use Doctrine\DBAL\Types\Type;
+use Illuminate\Database\Schema\Builder as SchemaBuilder;
 use Winter\Storm\Database\MemoryCache;
 use Winter\Storm\Database\QueryBuilder;
-use Illuminate\Database\Connection as ConnectionBase;
+use RuntimeException;
 
-class Connection extends ConnectionBase
+trait Connection
```

Comment:
Why was this changed to a trait from a class? It might make more sense to move the overrides into a new trait and keep this as a class that only implements the trait, just in case anyone was directly referencing it before. We can mark the class deprecated though.

Reply:
done